examples/official-storybook/stories/demo/typed-button.stories.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { ComponentProps } from 'react';
+import { Meta, Story } from '@storybook/react/types-6-0';
+import TsButton from '../../components/TsButton';
+
+export default {
+  title: 'Other/Demo/TsButton',
+  component: TsButton,
+  decorators: [
+    (StoryFn) => (
+      <>
+        <StoryFn />
```

Comment:
Would this also work with `{StoryFn()}` ? I don't know exactly why two different ways could be used but I assume both should work with no type issues?

Reply:
Yep tested